Dockerfile

```diff
@@ -26,7 +26,7 @@ ENV NODE_ENV=production \
 RUN yarn build
 
 # Backend builder stage
-FROM docker.io/library/golang:1.22.6-alpine3.20 as go-builder
+FROM docker.io/library/golang:1.23.2-alpine3.20 as go-builder
```

Comment:
just wondering: does `update-versions.py` work with this kind of version number as well? 
there's no micro release number for alpine